shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/model/Dataset.java

```diff
@@ -171,6 +171,8 @@ public abstract class Dataset extends AbstractEntity {
 	@Transient
 	public String SOPInstanceUID;
 
+	private String copyMessage;
```

Comment:
What kind of message can be set here ?

Reply:
Those are the possible values of copyMessage:

"This dataset is the copy of :"
"This dataset has been copied :"
"This acquisition is the copy of :"
"This acquisition has been copied :"
"This examination is the copy of :"
"This examination has been copied :"